docs/INSTALLATION.md

```diff
@@ -1,15 +1,50 @@
 # Installation
 
-<!-- TODO: write document
+## Windows users
 
-  This document should describe what the student needs to install
-  to allow working on the track on their local system using the CLI.
+Please install [Windows Subsystem for Linux](https://learn.microsoft.com/en-us/windows/wsl/install) and see the next section.
 
-  You can include the installation instructions in this document, but
-  usually it is better to link to a resource with the official installation
-  instructions, to prevent the instructions from becoming outdated.
+## Linux/Mac users
 
-  The contents of this document are displayed on the track's documentation
-  page at `https://exercism.org/docs/tracks/<track>/installation`.
+### Installing with brew
 
-  See https://exercism.org/docs/building/tracks/docs for more information. -->
+This is the easiest way to install Chapel. It will give you a basic working installation. This is enough for Exercism, but it has a few limitations, for example it does not support multi-locale. More advanced users may want to build Chapel from source code (next section).
```

Comment:
I had to look up what `multi-locale` is.

If you install via brew it should be easy to uninstall and install from source code later, if you decide you want to, right? Because if so I think we should take a more opinionated approach and say something like:

> This is the easiest way to install Chapel.
> It will give you a basic working installation, which is suitable for working through the Exercism track.
> More advanced users may want to build Chapel from source code (next section).

Reply:
this reads much better, I'll update